index.js

```diff
@@ -1,253 +1,38 @@
 
-var fs = require('fs'),
-    levelup = require('levelup'),
-    multilevel = require('level-multiply'),
-    parser = require('osm-pbf-parser')(),
-    through = require('through2'),
-    suggester = require('pelias-suggester-pipeline'),
-    propStream = require('prop-stream'),
-    settings = require('pelias-config').generate(),
-    dbclient = require('pelias-dbclient')(),
-    OsmiumStream = require('osmium-stream');
-
-var Backend = require('geopipes-elasticsearch-backend');
-
-// esclient.livestats();
+/**
+  The openstreetmap importer provides a way of parsing, mapping and
+  augmenting OSM data in to elasticsearch.
+**/
 
-var backend = function( index, type ){
-  return new Backend( dbclient.client, index, type );
+var elasticsearch = require('pelias-dbclient'),
+    adminLookup = require('pelias-admin-lookup'),
+    suggester = require('pelias-suggester-pipeline'),
+    dbmapper = require('./stream/dbmapper');
```

Comment:
There are pros/cons to this approach, the `dbmapper` is a pretty generic stream which is used to map `pelias.model.Document` objects to the syntax that is accepted by `pelias/dbclient`.

Exporting it makes it easier for 3rd party consumers to do that simple mapping **but** it strictly targets the `pelias` index, meaning consumers cannot build an index of another name.

I would love to see this functionality abstracted out to either `pelias/dbclient`, `pelias/model` or another module which is responsible for this domain and re-usable by other imports, for that reason I chose not to export the stream in order to avoid deprecating that API in the future.